resolwe_bio/docker_images/rnaseq/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM docker.io/resolwe/base:ubuntu-16.04
+
+LABEL MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends software-properties-common && \
```

Comment:
Check if you need 'software-properties-common'. I think we needed this to build / install bamliquidator in chipseq docker image. It might not be needed here